### ISSUE TITLE
Missing data

### ISSUE THREAD

**Issue by rugk** [rugk/crops-parser#16]
[In the list](https://github.com/rugk/crops-parser/blob/master/osmcrops.csv) everything marked with "???????" has an OSM tag, but we have no data for it.
I think we can likely ignore it for now, as I trust the FAO data quite much. I mean if blackberry is not produced and there are no statistics for it, then we cannot do anything and maybe there are really no orchards with them.

Also we have no data for tree nurseries (a thing to re-tag) as the FAO of course does not includes this (it's neither agriculture nor food).
We could integrate this data or… well… leave it out for now.

**Comment by westnordost** [comment]
Yeah well, I guess if these ?????? trees do not occur in the FAO data, it means they are cultivated not very much. There are thousands of plants, it is a matter of course that only the most important ones can be included. 

Keep in mind that it is work to create each tree defintion:
1. research, gather data
2. translate into all languages and search image
3. put into code

That cannot be done for each and every crop plant that exists on this planet.

For anything else, there is always the "Leave Note" button.

**Comment by rugk** [comment]
Right, although I'd say we do not need a translation for every crop in *every* language – just in the languages spoken in the countries, where this is in the top-5… (And if there are tourists, well…, nevermind, they either see the crop name somewhere written in the local language or know it in English, IMHO)

As you see in the OP, this issue is mostly about tree nurseries, you've mentioned once. These may belong to "orchard" too (Or don't?). At least they can be confused on satellite images. I mean who knows whether this is an apple tree or another one.

**Comment by westnordost** [comment]
The translations work like this: You create the base string in English. Then, for every language for which StreetComplete is translated this base string is marked as to be translated. You cannot selectively say that this or that crop should only be translated in XYZ.

https://poeditor.com/projects/view?id=97843

**Comment by westnordost** [comment]
Tree nurseries do not belong to orchard ([landuse=plant_nursery](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dplant_nursery)) but may look like orchards (and the other way round).

**Comment by rugk** [comment]
> You cannot selectively say that this or that crop should only be translated in XYZ.

Can't you add some notes? Or, at least, a general guide. I mean if contributors want to translate it, all right. It should fallback to English through. And maybe translators should know that the local grown fruits are more important to translate.
All in all, I think, translation is not an issue we should/need to care about. And if it's really needed, a simply (automated) dictionary lookup does it. You could even use Google translate, for one word, it is likely correct.

> https://poeditor.com/projects/view?id=97843

Yeah, in case you did not notice: I already joined.

> Tree nurseries do not belong to orchard (landuse=plant_nursery) but may look like orchards (and the other way round).

That's the point. Judging from our [criteria](https://github.com/rugk/crops-parser/wiki/Inclusion-conditions) we would have to include them. We just have no data. So this issue can be closed, if added for this is added.

**Comment by westnordost** [comment]
Yes, it always falls back to English. Or, since Android 7, to the language of your choice.

 No, I can't add notes.
